### PR TITLE
feat: routes fix

### LINKED ISSUE #285: Fix routing
**Describe the feature**

Remover o path /app/ da home. O link final deve ficar algo como: petdex.com/ e não petdex.com/app/

<details open>
  <summary>
    <b>Description</b>
  </summary>
  [Describe the new feature requested.]
</details>

<br>

<details> 
  <summary>
    <b>Use Case</b>
  </summary>
  
[Explain the use for this feature and how it might benefits the project.]
</details>

<br>

<details> 
  <summary>
    <b>Implementation Details</b>
  </summary>

[Provide any details or suggestions on how this feature could be implemented.]

</details>

<br>

<details> 
  <summary>
    <b>Visual Concepts</b>
  </summary>

[Include any visual representations or concepts if those are available and applicable.]

</details>

<br>

<details> 
  <summary>
    <b>Additional Information</b>
  </summary>
  
[Provide any additional information, such as context that might be relevant to the implementation of this specific feature.]
</details>

### PR DESCRIPTION
**Pull request by RafaelLimaC**
Closes #285 

<details open> 
  <summary>
    <b>Feature</b>
  </summary>

N/A

</details>

<details open> 
  <summary>
    <b>Bugfix</b>
  </summary>

- **Description**
  N/A

- **Cause**
  N/A

- **Solution**
N/A
</details>

<details> 
  <summary>
    <b>Changelog</b>
  </summary>
N/A
</details>

<details open> 
  <summary>
    <b>Visual evidences :framed_picture:</b>
  </summary>

</details>

<details open> 
  <summary>
    <b>Checklist</b>
  </summary>

- [ ] Issue linked
- [ ] Build working correctly
- [ ] Tests created
</details>

<details> 
  <summary>
    <b>Additional info</b>
  </summary>
N/A
</details>

### CODE REVIEW

File: src/layouts/app/components/SideMenu/index.js [hunk 1]
```diff
@@ -31,12 +31,12 @@ const html = `
...
+          <li><a data-vanilla-route-link="spa" class="side-menu-content__menuitens" href="/"><img src="${meusPets}" alt="Meus Pets">Meus Pets</a></li>
+          <li><a data-vanilla-route-link="spa" class="side-menu-content__menuitens" href="/pet-dex"><img src="${petdex}" alt="Pet Dex">Pet Dex</a></li>
+          <li><a data-vanilla-route-link="spa" class="side-menu-content__menuitens" href="/donates"><img src="${doacoes}" alt="Doacoes">Doações</a></li>
           <hr class="side-menu-content__lineinside"/>
-          <li><a data-vanilla-route-link="spa" class="side-menu-content__menuitens" href="/app/account"><img src="${conta}" alt="conta">Conta</a></li>
-          <li><a data-vanilla-route-link="spa" class="side-menu-content__menuitens" href="/app/settings"><img src="${configuracoes}" alt="configuracoes">Configurações</a></li>
+          <li><a data-vanilla-route-link="spa" class="side-menu-content__menuitens" href="/account"><img src="${conta}" alt="conta">Conta</a></li>
+          <li><a data-vanilla-route-link="spa" class="side-menu-content__menuitens" href="/settings"><img src="${configuracoes}" alt="configuracoes">Configurações</a></li>
```
Comment (juliaam) [inline src/layouts/app/components/SideMenu/index.js:39]: tem que ajustar a formatação

**Review by juliaam — CHANGES_REQUESTED** [review]

File: src/layouts/app/components/SideMenu/index.js (hunk 1 above)
Comment (juliaam) [inline src/layouts/app/components/SideMenu/index.js:39]: só ajustaria a formatação

Comment (juliaam) [inline src/layouts/app/components/SideMenu/index.js:39]:
```
<li>
  <a data-vanilla-route-link="spa" class="side-menu-content__menuitens" href="/">
    <img src="${meusPets}" alt="Meus Pets">Meus Pets
  </a>
</li>
```

**Review by juliaam — APPROVED** [review]

**Review by DominMFD — APPROVED** [review]

**Review by diogocaronte — APPROVED** [review]

**Review by Alecell — CHANGES_REQUESTED** [review]
Só deixa pronto um exemplo do create accont pf!

**Review by Luiginicoletti — APPROVED** [review]
Po so ta faltando uma descricao do PR pra saber o que foi feito hahahah

**Pull request closed without merging**